### DISCUSSION TITLE
Recursive parent child until criteria is met

### DISCUSSION THREAD

**Discussion by StanMalone** [Budibase/budibase#10326] [Help]
How can we develop a recursive function to establish parent-child relationships until a defined termination condition is met, specifically using Budibase's internal CouchDB database? Our objective is to generate a nested list with numbering, following the pattern of
1.
1.1.
2.
2.1.
2.1.1
3.
3.1.
3.1.1.
3.2.
4.

**Comment by andz-bb** [comment]
hey @StanMalone, I'm having some trouble understanding what you plan to generate this based on. Could you give an example of the input and how you would expect the data to look after the function runs? Thanks
